src/many/src/cose_helpers.rs

```diff
@@ -5,34 +5,84 @@ use coset::{
 };
 use std::collections::{BTreeMap, BTreeSet};
 
+/// Build an EdDSA CoseKey
+///
+/// # Arguments
+///
+/// * `x` - Public key
+/// * `d` - Private key
+pub fn eddsa_cose_key(x: Option<Value>, d: Option<Value>) -> CoseKey {
```

Comment:
There should always be a public key. The private key is optional.

Reply:
Fixed